drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -54,7 +54,7 @@ static void config_bus_clk_init(LL_UTILS_ClkInitTypeDef *clk_init)
 	clk_init->APB1CLKDivider = apb1_prescaler(
 					CONFIG_CLOCK_STM32_APB1_PRESCALER);
 
-#ifndef CONFIG_SOC_SERIES_STM32F0X
+#if !defined (CONFIG_SOC_SERIES_STM32F0X) && !defined (CONFIG_SOC_SERIES_STM32G0X)
```

Comment:
If APB2 not supported, you also need to update:
`config CLOCK_STM32_APB2_PRESCALER` in `drivers/clock_control/Kconfig.stm32`

Reply:
This comment still stands